prow/config.yaml

```diff
@@ -72,26 +72,17 @@ presubmits:
         - "--repo=github.com/kubernetes-incubator/$(REPO_NAME)=$(PULL_REFS)"
         - "--root=/go/src"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
-        - "--clean"
```

Comment:
Is the `clean` needed?

Reply:
hummm it's mostly for git cache, since you are not using it I think we can remove it.